coop_shift/__init__.py

```diff
@@ -1,4 +1,3 @@
-# -*- encoding: utf-8 -*-
```

Comment:
Please remove this long header from all file and replace by this style:

![image](https://user-images.githubusercontent.com/1914185/73592483-4ac31380-44fb-11ea-8c5f-c9607f7edb17.png)


Don't forget to add 
```
# Copyright (C) 2020-Today: Druidoo (https://www.druidoo.io)
```

And make sure you keep the copyrights and authors present in the previous header block